tools/test.sh

```diff
@@ -19,9 +19,28 @@ function prepare {
   # Webpack with the proto loader isn't used when running the tests, so the proto files need to be prepared manually
   pbjs -t json "webknossos-datastore/proto/SkeletonTracing.proto" > "$testBundlePath/SkeletonTracing.proto.json"
   pbjs -t json "webknossos-datastore/proto/VolumeTracing.proto" > "$testBundlePath/VolumeTracing.proto.json"
-  # --copy-files will copy files that are present in the source dir but are not transpiled (e.g.: json files)
-  # "$@" inside this function refers to all arguments of the prepare function, not all arguments of this bash script
-  BABEL_ENV=test babel --extensions .ts,.tsx "$jsPath" --out-dir "$testBundlePath" --copy-files "$@"
+
+  node_modules/.bin/esbuild \
+    --platform=node \
+    --format=cjs \
+    --define:process.env.BABEL_ENV=\"test\" \
+    --target=node10.4 \
+    --outdir="$testBundlePath" $($FIND frontend/javascripts \( -name "*.ts" -o -name "*.tsx" \))
+
+  # Copy files which were not compiled by esbuild (e.g., snapshots).
```

Comment:
This is an impressive command, but also slightly intimidating. Could you add an example file path before and after the transformation. This should make it easier to adapt the command in the future, in case the folder structure changes.

Reply:
Good point! I just extended the comment :)